roles/nvidia-network-operator/vars/main.yaml

```diff
@@ -0,0 +1,32 @@
+---
+#
+# variables for VFs, resource names and interface names
+# pf_name is the physical interface with sriov enabled
+# res_name must match container's network resources in job file 
+# if_name must match k8s network annotation name
+#
+num_vf: 8
+vendor_id: "15b3"
+link_type: "ib"
+mtu: 4096
+
+intf_resources:
```

Comment:
Will these apply for all hardware types or is it specific to DGX systems?

Reply:
This applies to all hardware types.